python/postprocessing/framework/output.py

```diff
@@ -7,48 +7,58 @@
 _rootBranchType2PythonArray = { 'b':'B', 'B':'b', 'i':'I', 'I':'i', 'F':'f', 'D':'d', 'l':'L', 'L':'l', 'O':'B' }
 
 class OutputBranch:
-    def __init__(self, tree, name, rootBranchType, n=1, lenVar=None, title=None, limitedPrecision=False):
+    def __init__(self, tree, name, rootBranchType, n=1, lenVar=None, title=None, limitedPrecision=None, saveBranch=True):
         n = int(n)
         self.buff   = array(_rootBranchType2PythonArray[rootBranchType], n*[0. if rootBranchType in 'FD' else 0])
         self.lenVar = lenVar
         self.n = n
-        self.precision = ROOT.ReduceMantissaToNbitsRounding(limitedPrecision) if limitedPrecision and rootBranchType=='F' else lambda x : x
+        self.precision = ROOT.ReduceMantissaToNbitsRounding(int(limitedPrecision)) if limitedPrecision and rootBranchType=='F' else lambda x : x
```

Comment:
Since the limitedPrecision is set to None by default, I think `if limitedPrecision is not None" here would be safer

Reply:
Done